package.json

```diff
@@ -20,29 +20,27 @@
 	"license": "GPL-3.0+",
 	"private": true,
 	"scripts": {
-		"start": "/bin/sh -c 'cd components/${1:-$0} && bolt start'",
+		"start": "/bin/sh -c 'cd components/${1:-$0} && yarn start'",
 		"docs": "docz dev",
-		"docs:build": "docz build && bolt docs:redirects",
+		"docs:build": "docz build && yarn docs:redirects",
 		"docs:redirects": "cp helpers/public/* docs/",
-		"nuke": "rm -rf node_modules && bolt ws exec -- rm -rf node_modules",
-		"build": "bolt ws exec -- rm -rf dist && bolt ws exec -- bolt build",
-		"fresh": "bolt nuke && bolt install",
+		"nuke": "find ./components -type d -name 'node_modules' -exec rm -rf {} + && find ./brands -type d -name 'node_modules' -exec rm -rf {} +",
+		"build": "node ./helpers/buildBrands.js",
```

Comment:
```suggestion
		"build": "preconstruct build && node ./helpers/buildBrands.js",
```

Reply:
`preconstruct build` exposes a few issues with the repo - I have fixed a couple but will remove that for now, noting that this error needs to be fixed:

```🎁 error @westpac/alert { TypeError: Cannot read property 'id' of null
🎁 error @westpac/alert     at Object.resolveId (/Users/simonswiss/Sites/thinkmill/westpac/GEL/node_modules/preconstruct/cli/dist/preconstruct.cjs.dev.js:1055:20)
🎁 error @westpac/alert     at <anonymous>
🎁 error @westpac/alert   code: 'PLUGIN_ERROR',
🎁 error @westpac/alert   plugin: 'flow-and-prod-dev-entry',
🎁 error @westpac/alert   hook: 'resolveId' }```